### DISCUSSION TITLE
How do I use Clang for MSVC with vcpkg?

### DISCUSSION THREAD

**Discussion by kkqy** [microsoft/vcpkg#26673] [Q&A] [answered]
I installed LLVM/Clang for MSVC,but it seems like it can't find header files or lib files installed by vcpkg.
```
PS D:\temp> clang .\test.c  
.\test.c:1:10: fatal error: 'iconv.h' file not found
#include <iconv.h>
         ^~~~~~~~~
1 error generated.
```
I have installed libiconv succesfully, and it can be used in MSVC, but clang can't find it.

**Comment by Neumann-A** [comment, marked as answer]
If you are invoking the compiler by hand you are responsible for setting all necessary paths/compile options. So unless you start using a buildsystem there is nothing vcpkg can do.
